File: C/bee1008.c
#include <stdio.h>
#define length 3
int main(){
    double array[length];
    double salary;
    for (int i = 0;i<length;i++){
        scanf("%lf",&array[i]);
    }
    salary = array[1]*array[2];
    printf("NUMBER = %.0f\nSALARY = U$ %.2f\n", array[0],salary);
}

File: js/bee1008.js
var input = require('fs').readFileSync('js/stdin', 'utf8');
var lines = input.split('\n');

var number = lines[0]
var hours = parseInt(lines[1])
var perhour = parseFloat(lines[2])

var salary = (hours*perhour)
console.log("NUMBER = "+number)
console.log(`SALARY = U$ ${salary.toFixed(2)}`)